iis/mymodule.cpp

```diff
@@ -808,6 +810,14 @@ CMyHttpModule::OnBeginRequest(IHttpContext* httpContext, IHttpEventProvider* pro
         return RQ_NOTIFICATION_CONTINUE;
     }
 
+    // If we previously failed to load the config, don't spam the event log by trying and failing again
+    if (config->configLoadingFailed)
+    {
+        httpContext->GetResponse()->SetStatus(500, "WAF internal error");
```

Comment:
And here?

Perhaps, we could use a common string literal in all places where we set the status for invalid configuration.

Reply:
used your other suggestion on the lambda